app/src/main/java/edu/uc/reedws/musiclink/ui/main/MainFragment.kt

```diff
@@ -10,10 +10,6 @@ import edu.uc.reedws.musiclink.R
 
 class MainFragment : Fragment() {
 
-    companion object {
-        fun newInstance() = MainFragment()
-    }
-
```

Comment:
I removed the companion object from the top because I learned from one of the professor's lecture videos (Sprint 1 code review) that companion object should be down below in the class as it is one of the Kotlin coding conventions.